src/components/molecules/TableHeader/TableHeader.tsx

```diff
@@ -139,19 +135,7 @@ const TableHeader: React.FC<TableHeaderProps> = ({
   return (
     <div className="row TableHeader">
       <div className="TableHeader__leave-table">
-        <button
-          type="button"
-          title={`Leave ${seatedAtTable}`}
-          className="TableHeader__leave-button"
-          onClick={leaveSeat}
-        >
-          <FontAwesomeIcon
-            className="TableHeader__leave-button--icon"
-            icon={faChevronLeft}
-            size="xs"
-          />
-          Leave table
-        </button>
+        <BackButton title={"Leave table"} onClick={leaveSeat} />
```

Comment:
Should the title be `Leave ${seatedAtTable}` as per the old code? Or is this change from a new design?

Reply:
And if it is correct as is, then there is no need to include the `{` `}` around a plain string:
```suggestion
        <BackButton title="Leave table" onClick={leaveSeat} />
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1492/commits/516bcef90616e538cbf0a8609a96d134bfbf4fcc